### PR TITLE
Navbar tidy up

### PR DESCRIPTION
**Pull request by oherst**
This PR updates the brunch config to keep the app and homepage css separate (and converts app.css to app.scss).

I've also tidied up the navbar and add some basic styles :)

screen shot 2017-10-31 at 22 48 55

### CODE REVIEW

File: assets/css/app.scss [hunk 1]
```diff
@@ -28,3 +28,10 @@
   border: 1px solid #d8cccc;
   border-radius: 5px;
 }
+
+.navbar-brand {
+  font-family: 'Oswald', sans-serif;
+}
+.navbar-custom-bg {
+  background-color: #663399;
```
Comment (efexen) [inline assets/css/app.scss]: Could we declare this as a variable either at the top of `app.scss` or in a new `colors.scss` which is then imported?

**Review by efexen — COMMENTED** [review]
Thanks Ollie this is great! I've left one small recommendation

**Comment by oherst** [comment]
Done 👍

**Comment by efexen** [comment]
Awesome thank you 💖

**Pull request merged**